### DISCUSSION TITLE
Builds stuck on "Queued" with no blocking builds

### DISCUSSION THREAD

**Discussion by gwkline** [vercel/community#5871] [Help]
### Summary

Hi there. As of about 30 mins ago, builds for our NextJS project have been stuck on queued. We are on the hobby plan, so only 1 concurrent build, but as of right now there are no builds that are "Building".
![CleanShot 2024-02-19 at 13 15 11@2x](https://github.com/vercel/community/assets/77706736/ed2ba2ad-fd5c-42b3-a1b5-0cbcb99edc11)

Any idea how this can be resolved? Thanks.

### Steps to Reproduce

N/A

**Comment by sreedhap** [comment]
same issue here, we are on Enterprise plan, and no other parallel builds at the moment

**Comment by ArchangelGCA** [comment]
Same here, Hobby plan.
Tried already twice to re-deploy, it's about half an hour now.

**Comment by 2ssue** [comment]
Same issue here, Pro plan. We have 2 slots & no other parallel builds for 1 hour.

**Comment by edlefebvre** [comment]
Same here, Hobby plan.

**Reply by edlefebvre** [reply]
For info, on vercel status : 

[Elevated Build Queue Times](https://www.vercel-status.com/incidents/s8d0zc1j7c7x)
Identified - The issue has been identified and a fix is in the works. To note: this issue is only present on builds using Node 20. Builds using Node 18 and below are functioning normally.
Feb 19, 2024 - 19:13 UTC
Investigating - We are currently investigating reports of queued builds. We will share updates as they become available.
Feb 19, 2024 - 19:01 UTC

**Comment by gwkline** [comment]
Seems to be resolved for me, if anyone else can confirm feel free to close this discussion.

**Reply by onexbash** [reply]
@gwkline Can confirm. Also had this issue today, but seems to be fixed since ~30mins.

**Reply by sreedhap** [reply]
Yep, builds are working again!

**Reply by ArchangelGCA** [reply]
Confirm
